### ISSUE TITLE
con_log_color_warning is temporarily ignored

### ISSUE THREAD

**Issue by hexaflexahexagon** [momentum-mod/game#1420]
**Describe the bug**
The picture below shows a console readout right after booting the game. You can see the warnings above that are orange (the default 255 140 0 color) and then the ones after I gain control that are a pink-ish red (226 84 84), what I actually set.
![image](https://user-images.githubusercontent.com/62961532/124608361-aace8f80-de3c-11eb-83e0-c04af81f56d1.png)

**To Reproduce**
Boot the game with any non-default `log_con_color_warning` command value

**Expected behavior**
The value set in `log_con_color_warning` should always be used

**Desktop (*please complete the following information*):**
Windows, Dx9, VGUI

**Comment by lvaness** [comment]
Hmm, that would probably be a pretty complicated fix with how ConVars get initialized, I'd say this is maybe something to consider for a Panorama console

**Comment by Gocnak** [comment]
Yeah archived (config.cfg) values are only read after a lot of core stuff starts up, meaning the color is most likely going to be false for a while.

**Comment by hexaflexahexagon** [comment]
Yeah that's about what I expected. we were talking about eventually doing syntax highlighting perhaps so this could be revisited then (in panorama). It also wouldn't be an issue if the new default orange wasn't so off-putting and different from the old default :p

**Comment by lvaness** [comment]
This will be fixed by initializing the panel much later (which will still print all relevant messages as they get queued)

**Comment by VortexParadox** [comment]
<img src=https://user-images.githubusercontent.com/35622748/135745418-e2b330b6-9f36-434e-afcf-6194e4ad3c86.png width=400>
Seems like it's not fixed yet?

**Comment by lvaness** [comment]
Yeah i'd hoped that would be resolved but sadly it didn't work like i thought. There might be an internal change at some point fixing this, though

**Comment by lvaness** [comment]
This is definitely doable now with the latest console changes + some internal changes necessary
